topics/epigenetics/tutorials/formation_of_super-structures_on_xi/tutorial.md

```diff
@@ -306,7 +307,7 @@ Since in this tutorial we are interested in assessing H3K4me3, H3K27me3 and CTCF
 >    ```
 >
 > 3. Rename the files
-> 3. **multiBamSummary** {% icon tool %} with the following parameters
+> 3. {% tool [multiBamSummary](toolshed.g2.bx.psu.edu/repos/bgruening/deeptools_multi_bam_summary/deeptools_multi_bam_summary/3.5.1.0.0) %} with the following parameters:
```

Comment:
Can we use the latest version of the tool which is: Galaxy Version 3.5.2+galaxy0 or is there a specific reason to use one prior version?

Reply:
because this is the most common latest version on .org and .eu